.github/workflows/ci-unix-static-sanitized.yml

```diff
@@ -0,0 +1,75 @@
+name: CI Unix Static Sanitized
+on: [ push, pull_request ]
+
+permissions:
+  contents: read
+
+jobs:
+  build-static-sanitized:
+    runs-on: ubuntu-latest
```

Comment:
Thanks for adding this CI workflow. If this works on macos-latest, it would be good to add it.

Reply:
Done in #2061